tex/texlive-bin/Portfile

```diff
@@ -253,6 +253,20 @@ if {[string match *clang* ${configure.cxx}]} {
     configure.cxxflags-append -Wno-reserved-user-defined-literal
 }
 
+destroot.violate_mtree  yes
+
+# Useful variables to set up the TeXDist directory structure.
+# See below in post-destroot and pre-activate.
+set td_root         /Library/TeX/Distributions
+set td_fd           /.FactoryDefaults
+if {[string equal ${prefix} {/opt/local}]} {
+    set td_name     MacPorts-TeXLive
+} else {
+    regsub -all {/} [string trim ${prefix} {/}] {_} prefix_
+    set td_name     MacPorts-${prefix_}-TeXLive
```

Comment:
The prefix could potentially go to the end, but I have no strong opinion about this. The idea to add a prefix makes sense in any case.